lib/add-tests.coffee

```diff
@@ -52,6 +52,49 @@ addTests = (raml, tests, hooks, parent, callback, testFactory) ->
     # In case of issue #8, resource does not define methods
     resource.methods ?= []
 
+    if sortFirst && resource.methods.length > 1
```

Comment:
Think large chunks of **abao** could stand redesign. But I'd prefer having the functionality over refactoring it first. Would rather do suggested refactoring as different issue.